spec/index.html

```diff
@@ -1041,6 +1063,16 @@ <h3>Algorithm</h3>
               <a>input dataset</a> with both their original identifiers,
               and associated canonical identifiers.</p>
           </details>
+          <div class="issue" data-number="155">
```

Comment:
My proposal is to remove the issue reference note altogether. We will have to remove it before CR for the reason I already stated, and I do not expect this issue to be fully resolved before TPAC anyway. It does not really say more than what is in the intro paragraph anyway.